deps/exokit-bindings/webglcontext/src/webgl.cc

```diff
@@ -4604,14 +4593,10 @@ NAN_METHOD(WebGLRenderingContext::GetParameter) {
       GLint numFormats;
       glGetIntegerv(GL_NUM_COMPRESSED_TEXTURE_FORMATS, &numFormats);
 
-      unique_ptr<GLint[]> params(new GLint[numFormats]);
-      glGetIntegerv(name, params.get());
-
-      Local<Array> arr = Nan::New<Array>(numFormats);
-      for (size_t i = 0; i < numFormats; i++) {
-        arr->Set(i, JS_INT(params[i]));
-      }
-      info.GetReturnValue().Set(arr);
+      GLint* params = new GLint[numFormats];
```

Comment:
note that numFormats could be -1 here if glGetIntegerv returns an error (but this was true of the old code as well)

Reply:
Would that ever be a legit value, or only if we have a broken build/driver?